plugins/steelseries/fu-steelseries-hid.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2022 Gaël PORTAY <gael.portay@collabora.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#pragma once
+
+#include <fwupdplugin.h>
+
+#define FU_TYPE_STEELSERIES_HID (fu_steelseries_hid_get_type())
+
+#if 0
+#define __USE_HID_DEVICE__
+/*
+ * FIXME:
+ * 06:44:09:0074 FuUsbDevice          fu_usb_device_incorporate: assertion 'FU_IS_USB_DEVICE(donor)' failed
+ * 06:44:09:0074 GLib-GObject         invalid cast from 'FuSteelseriesHid' to 'FuUdevDevice'
+ * 06:44:09:0074 FuUdevDevice         fu_udev_device_set_physical_id: assertion 'FU_IS_UDEV_DEVICE(self)' failed
```

Comment:
If you want to be a FuHidDevice then you need to derive from FuUsbDevice.

Reply:
okay, thanks.